src/Components/Home/CategoryCard.jsx

```diff
@@ -0,0 +1,12 @@
+import { NavLink } from "react-router-dom"
+export default function ({ category }) {
+
+    return (
+        <NavLink to="/products" className="link">
+            <div className="card">
+                <img className="card-img" alt={category.categoryName} src={category.img} />
+                <p className="heading-4 l-sp-1 theme-shade-2">{category.categoryName}</p>
+            </div>
+        </NavLink>
```

Comment:
I think it'd be better to use `Link` instead of `NavLink` here, since `Navlinks` should be used for those links which are common among pages, but this categoryCard will only be shown in the home page

Reply:
Thanks for pointing it out , i will change it